lib/renderer.js

```diff
@@ -1421,6 +1417,7 @@
             } else /* if either a layerComp or the whole document */ {
                 if (layerComp) {
                     pixmapSettings.compId = layerComp.id;
+                    pixmapSettings.documentBounds = true;
```

Comment:
@pineapplespatula can you explain this line?  It seems to be a difference I wasn't expecting (vs. d9a0fa8)